### PR TITLE
Vhm aws permissions #14391 keichwa

### PR DESCRIPTION
**Pull request by keichwa**
# Description
https://github.com/SUSE/spacewalk/issues/14391

Short summary of why you created this PR

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [ ] Manager-4.2
- [ ] Manager-4.1
- [ ] Manager-4.0

# Links

Fixes #<insert issue or PR link, if any>

### CODE REVIEW

**Comment by keichwa** [comment]
@cbosdo can you please check whether I got it right?

**Comment by cbosdo** [comment]
@keichwa I have no clue about the VHM features even though it has the `virtualization` word in it

**Comment by keichwa** [comment]
> @keichwa I have no clue about the VHM features even though it has the `virtualization` word in it

ok, thanks for feedback!

@admd , I please need a dev who can check this PR.  It is about VHM and VHM AWS.

**Comment by admd** [comment]
@meaksh will be reviewing this PR, Karl.

**Review by meaksh — APPROVED** [review]
LGTM

File: modules/client-configuration/pages/vhm-aws.adoc [hunk 1]
```diff
@@ -40,3 +40,33 @@ Instances running on the Amazon public cloud report a UUID to the {productname}
 ----
 I1234567890abcdef0
 ----
+
+
+
+== AWS Permissions for Virtual Host Manager
+
+For security reasons, always grant the least privilege possible for a task to be performed.
+Thus using the Access Key for a user with excessive permissions to AWS is not advised.
```
Comment (jcayouette) [inline modules/client-configuration/pages/vhm-aws.adoc:49]: "Using an Access Key with excessive permissions for users connecting to AWS is not advised."

**Review by jcayouette — CHANGES_REQUESTED** [review]
Looks good other than the commented bit.

File: modules/client-configuration/pages/vhm-aws.adoc (hunk 1 above)
Comment (keichwa) [reply modules/client-configuration/pages/vhm-aws.adoc:49]:
```suggestion
Using an Access Key with excessive permissions for users connecting to AWS is not advised.
```

**Review by keichwa — COMMENTED** [review]
thank you, Joseph!

**Pull request merged**